doc/modules/ROOT/pages/tutorials/graph-analytics-serverless-standalone.adoc

```diff
@@ -0,0 +1,268 @@
+fruits_df = pd.DataFrame(
+    [
+        {"nodeId": 8, "name": "Apple", "tropical": 0, "sourness": 0.3, "sweetness": 0.6},
+        {"nodeId": 9, "name": "Banana", "tropical": 1, "sourness": 0.1, "sweetness": 0.9},
+        {"nodeId": 10, "name": "Mango", "tropical": 1, "sourness": 0.3, "sweetness": 1.0},
+        {"nodeId": 11, "name": "Plum", "tropical": 0, "sourness": 0.5, "sweetness": 0.8},
+    ]
+)
+fruits_df["labels"] = "Fruit"
+
+like_relationships = [(0, 8), (1, 9), (2, 10), (3, 10), (4, 9), (5, 11), (7, 11)]
+likes_df = pd.DataFrame([{"sourceNodeId": src, "targetNodeId": trg} for (src, trg) in like_relationships])
+likes_df["relationshipType"] = "LIKES"
+
+knows_relationship = [(0, 1), (0, 2), (1, 2), (1, 3), (1, 4), (2, 5), (7, 3)]
+knows_df = pd.DataFrame([{"sourceNodeId": src, "targetNodeId": trg} for (src, trg) in knows_relationship])
+knows_df["relationshipType"] = "KNOWS"
+----
+
+== Construct Graph from DataFrames
+
+Now that we have imported a graph to our database, we create graphs
+directly from pandas `DataFrame` objects. We do that by using the
+`gds.graph.construct()` method.
+
+[source, python, role=no-test]
+----
+# Dropping `name` column as GDS does not support string properties
+nodes = [people_df.drop(columns="name"), fruits_df.drop(columns="name")]
+relationships = [likes_df, knows_df]
+
+G = gds.graph.construct("people-fruits", nodes, relationships)
+str(G)
+----
+
+== Running Algorithms
+
+We can now run algorithms on the constructed graph. This is done using
+the standard GDS Python Client API. There are many other tutorials
+covering some interesting things we can do at this step, so we will keep
+it rather brief here.
+
+We will simply run PageRank and FastRP on the graph.
+
+[source, python, role=no-test]
+----
+print("Running PageRank ...")
+pr_result = gds.pageRank.mutate(G, mutateProperty="pagerank")
+print(f"Compute millis: {pr_result['computeMillis']}")
+print(f"Node properties written: {pr_result['nodePropertiesWritten']}")
+print(f"Centrality distribution: {pr_result['centralityDistribution']}")
+
+print("Running FastRP ...")
+frp_result = gds.fastRP.mutate(
+    G,
+    mutateProperty="fastRP",
+    embeddingDimension=8,
+    featureProperties=["pagerank"],
+    propertyRatio=0.2,
+    nodeSelfInfluence=0.2,
+)
+print(f"Compute millis: {frp_result['computeMillis']}")
+# stream back the results
+result = gds.graph.nodeProperties.stream(G, ["pagerank", "fastRP"], separate_property_columns=True)
+
+result
+----
+
+To resolve each `nodeId` to name, we can merge it back with the source
+data frames.
+
+[source, python, role=no-test]
+----
+names = pd.concat([people_df, fruits_df])[["nodeId", "name"]]
+result.merge(names, how="left")
+----
+
+== Deleting the session
+
+Now that we have finished our analysis, we can delete the session. The
+results that we produced were written back to our Neo4j database, and
+will not be lost. If we computed additional things that we did not write
+back, those will be lost.
```

Comment:
Maybe I am not seeing it, but where is it that results are being written back to the DB?

Reply:
there is no write-back to the db here. just forgot to remove this part